demisto_sdk/commands/common/tools.py

```diff
@@ -1801,6 +1799,40 @@ def find_type(
     Returns:
         FileType | None: Enum representation of the content file type, None otherwise.
     """
+    from demisto_sdk.commands.content_graph.objects import (
```

Comment:
isn't it better to have map between `ContentType` and `FileType` and call `ContentType.by_schame`?